backend/dal.keycloak/Partials/PimsKeycloakUserService.cs

```diff
@@ -160,6 +162,51 @@ public async Task<Entity.User> UpdateUserAsync(Entity.User user)
 
             return euser;
         }
+
+        /// <summary>
+        /// Updates the specified access request in the datasource. if the request is granted, update the associated user as well.
+        /// </summary>
+        /// <param name="entity"></param>
+        /// <exception type="KeyNotFoundException">Entity does not exist in the datasource.</exception>
+        /// <returns></returns>
+        public async Task<Entity.AccessRequest> UpdateAccessRequest(Entity.AccessRequest entity)
```

Comment:
Rename to `UpdateAccessRequestAsync` to keep in alignment with other async pattern.